README.md

```diff
@@ -14,7 +14,7 @@ measures over the output of a numerical routine, which then can be propagated al
 This repository aims to implement methods from PN in Python 3 and to provide a common interface for them. This is
 currently a work in progress, therefore interfaces are subject to change.
 
-## Installation and Documentation
+## Installation
```

Comment:
If this PR enables people to install ProbNum through `pip install probnum`, we should add such a line to the README (and possibly the documentation).